knowhere/knowhere/index/vector_index/impl/bruteforce/CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
 cmake_minimum_required(VERSION 3.10)
```

Comment:
please refer to nsg, we need not add bruteforce as a sub-project, so we need not add CMakeLists.txt here.

Reply:
I kept the CMakeLists.txt for bruteforce directory, so as to run demo tests.